dcicutils/misc_utils.py

```diff
@@ -999,3 +1029,8 @@ def delete(self, file):
         raise UncustomizedInstance(instance=thing, field=key)
     else:
         return value
+
+
+# Deprecated names, still supported for a while.
+HMS_TZ = REF_TZ
+hms_now = ref_now
```

Comment:
Is it convention to do this at the bottom of the file? I would think it makes sense to put these next to the respective definition ie: by `REF_TZ`.

Reply:
It's _a_ convention, but I'm not wedded to it if it bugs you. The idea is to say it's an after-the-fact thing to not mention these names anywhere in the basic file and only at the end. Putting them here puts them out of scope to refer to at top-level. Python makes it hard to put them out of scope from function definitions without putting them in another file, which wouldn't accomplish the purpose.

Reply:
I didn't get an answer on this, but I'm inclined to think my rationale is reasonable so I think I'll leave this as-is.